gateway-protocol/src/main/proto/gateway.proto

```diff
@@ -84,69 +84,69 @@ message CompleteJobRequest {
   // instance
   string bpmnProcessId = 2;
-  // the version of the workflow definition which was used to create the workflow instance
+  // the version of the process definition which was used to create the process instance
   int32 version = 3;
-  // the unique identifier of the created workflow instance; to be used wherever a request
-  // needs a workflow instance key (e.g. CancelWorkflowInstanceRequest)
-  int64 workflowInstanceKey = 4;
+  // the unique identifier of the created process instance; to be used wherever a request
+  // needs a process instance key (e.g. CancelProcessInstanceRequest)
+  int64 processInstanceKey = 4;
 }
 
-message CreateWorkflowInstanceWithResultRequest {
-  CreateWorkflowInstanceRequest request = 1;
-  // timeout (in ms). the request will be closed if the workflow is not completed
+message CreateProcessInstanceWithResultRequest {
+  CreateProcessInstanceRequest request = 1;
+  // timeout (in ms). the request will be closed if the process is not completed
   // before the requestTimeout.
   // if requestTimeout = 0, uses the generic requestTimeout configured in the gateway.
   int64 requestTimeout = 2;
-  // list of names of variables to be included in `CreateWorkflowInstanceWithResultResponse.variables`
+  // list of names of variables to be included in `CreateProcessInstanceWithResultResponse.variables`
   // if empty, all visible variables in the root scope will be returned.
   repeated string fetchVariables = 3;
 }
 
-message CreateWorkflowInstanceWithResultResponse {
-  // the key of the workflow definition which was used to create the workflow instance
-  int64 workflowKey = 1;
-  // the BPMN process ID of the workflow definition which was used to create the workflow
+message CreateProcessInstanceWithResultResponse {
+  // the key of the process definition which was used to create the process instance
+  int64 processKey = 1;
+  // the BPMN process ID of the process definition which was used to create the process
   // instance
   string bpmnProcessId = 2;
-  // the version of the workflow definition which was used to create the workflow instance
+  // the version of the process definition which was used to create the process instance
   int32 version = 3;
-  // the unique identifier of the created workflow instance; to be used wherever a request
-  // needs a workflow instance key (e.g. CancelWorkflowInstanceRequest)
-  int64 workflowInstanceKey = 4;
+  // the unique identifier of the created process instance; to be used wherever a request
+  // needs a process instance key (e.g. CancelProcessInstanceRequest)
+  int64 processInstanceKey = 4;
   // JSON document
   // consists of visible variables in the root scope
   string variables = 5;
 
 }
 
-message DeployWorkflowRequest {
-  // List of workflow resources to deploy
-  repeated WorkflowRequestObject workflows = 1;
+message DeployProcessRequest {
+  // List of process resources to deploy
+  repeated ProcessRequestObject processes = 1;
```

Comment:
We can think about renaming the field to `resources`. In the future, we might deploy also other things than just BPMNs (e.g. DMNs, form definitions, etc.).

Reply:
I imagine these would simply be different fields, as they might be of different type.

Reply:
I didn't touched it as I would wait for when we actually implement that feature